diff --git a/node_modules/react-native-nitro-modules/NitroModules.podspec b/node_modules/react-native-nitro-modules/NitroModules.podspec
index 2111b37..c7cef11 100644
--- a/node_modules/react-native-nitro-modules/NitroModules.podspec
+++ b/node_modules/react-native-nitro-modules/NitroModules.podspec
@@ -15,7 +15,7 @@ Pod::Spec.new do |s|
   s.source       = { :git => "https://github.com/mrousavy/nitro.git", :tag => "#{s.version}" }
   s.platforms    = {
     :ios => min_ios_version_supported,
-    :visionos => 1.0,
+    :visionos => '1.0',
     :macos => 10.13,
     :tvos => 13.4,
   }
